internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/AggPushStatusCleanUpStats.java

```diff
@@ -19,6 +19,11 @@ public AggPushStatusCleanUpStats(
         isUnregisterMetricForDeletedStoreEnabled);
   }
 
+  public void recordLeakedPushStatusCount(String storeName, int count) {
+    totalStats.recordLeakedPushStatusCount(count);
+    getStoreStats(storeName).recordLeakedPushStatusCount(count);
```

Comment:
Is it really useful to record this on per store level? I suppose we only need a total number + a controller log to know what resources get deleted...

Reply:
Good point! We don't really need store level metric here. Let me clean this up.